mautrix_telegram/portal.py

```diff
@@ -2505,6 +2537,18 @@ async def _handle_telegram_text(
         if is_bot and self.get_config("bot_messages_as_notices"):
             content.msgtype = MessageType.NOTICE
         await intent.set_typing(self.mxid, is_typing=False)
+
+        if (
+            getattr(evt, "media")
```

Comment:
getattr throws `AttributeError`s unless you provide a default, probably better to just use hasattr here

```suggestion
            hasattr(evt, "media")
```